extensions/wrapper/wrapper-ee-api/src/main/java/de/sovity/edc/ext/wrapper/api/ee/EnterpriseEditionResource.java

```diff
@@ -15,23 +15,121 @@
 
+import java.util.List;
+import java.util.Map;
+
 /**
  * Our EDC Enterprise Edition API Endpoints to be included in our generated EDC API Wrapper Clients
  */
 @Path("wrapper/ee")
 @Tag(name = "Enterprise Edition", description = "EDC Enterprise Edition API Endpoints. Included here for ease of use.")
 public interface EnterpriseEditionResource {
-
     @GET
     @Path("connector-limits")
     @Produces(MediaType.APPLICATION_JSON)
     @Operation(description = "Available and used resources of a connector.")
     ConnectorLimits connectorLimits();
+
+    @POST
+    @Path("file-storage/stored-files")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @Operation(summary = "Upload a file.",
+               description = "Upload a file to the file storage. <br> On a successful upload to the file storage " +
+                       "a StoredFile object is returned. <br> The assetProperties remain empty and are only added upon " +
+                       "a asset create request.")
+    @RequestBody(content =
+            @Content(mediaType = MediaType.MULTIPART_FORM_DATA,
+                     schema = @Schema(type = "object",
+                                      requiredProperties = {"fileName", "fileContent"}),
+                     schemaProperties = { @SchemaProperty(name = "fileName",
+                                                          schema = @Schema(type = "string",
+                                                                           format = "string",
+                                                                           example = "myFile.txt")
+                                                         ),
+                                          @SchemaProperty(name = "fileContent",
+                                                          schema = @Schema(type = "string",
+                                                                           format = "binary",
+                                                                           example = "fileContent")
+                                                         )
+                    }
+            )
+    )
+
+    @ApiResponses(value ={
+            @ApiResponse(responseCode = "200",
+                         description = "File successfully uploaded to the file storage.",
+                         content = @Content(schema = @Schema(implementation = StoredFile.class))
+            )}
+    )
+    StoredFile uploadStoredFile(
+            @Parameter(description = "The name of the file.",
+                       required = true,
+                       style = ParameterStyle.FORM
+            ) String fileName,
+
+            @Parameter(description = "The file content to upload.",
+                       required = true,
+                       style = ParameterStyle.FORM
+            ) byte[] fileContent
+    );
+
+    //ToDo: this API endpoint might be extended with query parameters in the future.
```

Comment:
I know, but this will remain on a feature branch until the full feature is ready to be merged in.   I added a ToDo here maybe because you would give the thumbs or down that we don't require this feature at the moment.

Reply:
I removed the ToDo, but would like to know your thoughts on my comment.